plugins/frontend/backstage-plugin-github-pull-requests/src/mocks/mocks.ts

```diff
@@ -1167,7 +1248,7 @@ export const yourOpenPullRequests = {
       closed_at: null,
       author_association: 'CONTRIBUTOR',
       active_lock_reason: null,
-      draft: true,
+      draft: false,

```

Comment:
The draft key here has no effect here being changed but both entries were drafts so it didn't really make sense (in terms of the "story" of the test) when compared to the draft PR mock just below